### ISSUE TITLE
Problem with pseudo-label generation

### ISSUE THREAD

**Issue by Rzy1221** [chaneyddtt/ScarceNet#7]
Dear author, when I run this command，Create folder root/data/pseudo_labels/5shots/ and generate pseudo labels by running:

CUDA_VISIBLE_DEVICES=0 python tools/train_mt_part.py --cfg experiments/ap10k/hrnet/w32_256x192_adam_lr1e-3.yaml --animalpose --generate_pseudol --pretrained output/output_animal_hrnet5_part/model_best.pth OUTPUT_DIR test MODEL.NAME pose_hrnet_part TRAIN.BATCH_SIZE_PER_GPU 32 GPUS [0,] LABEL_PER_CLASS 5

There's been an error. My CPU usage is high and seems to be stuck
![image](https://github.com/chaneyddtt/ScarceNet/assets/76418768/f83c882c-6928-4db7-82bc-cc68a5b967a6)

My first step of training with 5 images per species went well, but I got stuck generating false labels.
Looking forward to your answer.

**Comment by chaneyddtt** [comment]
Hi @Rzy1221, it is not stuck, it is generating pseudo labels and the process is quite slow.

**Comment by Rzy1221** [comment]
Thanks for your reply！ I got the result in a few longer hours.
![image](https://github.com/chaneyddtt/ScarceNet/assets/76418768/9ac8e948-1632-408e-b434-46cadffeb5f6)
